src/resources/locales/en.json

```diff
@@ -89,7 +89,9 @@
     "progressDownload": "Downloading",
     "endDownload": "Download completed",
     "failedDownload": "Download Failed",
-    "canceledDownload": "Download Canceled"
+    "canceledDownload": "Download Canceled",
+    "seeMore": "See more",
+    "seeLess": "See less"
```

Comment:
@panaC @manuellagrand do any of you speak basic German, to add "See more/less" to https://github.com/readium/readium-desktop/blob/develop/src/resources/locales/de.json ?

Reply:
@Scarsniik could you please add the German strings? Then we can merge this PR.